holidays/calendars.py

```diff
@@ -63,6 +63,37 @@ def _islamic_to_gre(g_year: int, h_month: int, h_day: int) -> Iterable[date]:
     return (gre_date for gre_date in gre_dates if gre_date.year == g_year)
 
 
+def _n_weekday_from_date(d: date, weekday: int, n: int = 1) -> date:
```

Comment:
I changed the args order too to better reflect the name:
```suggestion
def _get_nth_weekday_since(n: int,  weekday: int, since: date) -> date:
```

Reply:
Is it correct to use "since" for dates **before**?

Reply:
IMHO no. I suggest that the function be ``called _get_nth_weekday_from``, the argument called ``from``, and that the docstring address the option of a negative ``n``.

Reply:
Ooops...🤦

Sorry

Reply:
Changed args order, function renamed for now to `_get_nth_weekday_from`, date argument name still to be questionable.

Reply:
You can use `from_dt` or just `dt` then.